samples/Samples.AspNetCore3/Views/Shared/_Layout.cshtml

```diff
@@ -46,7 +46,7 @@
     @RenderSection("scripts", required: false)
 
     @* Simple options are exposed...or make a full options class for customizing. *@
-    <mini-profiler position="@RenderPosition.Right" max-traces="5" color-scheme="ColorScheme.Auto" />
+    <mini-profiler position="@RenderPosition.Right" max-traces="5" color-scheme="ColorScheme.Auto" nonce="45" />
```

Comment:
Given that nonce is supposed to be random on every page load, I'm not sure it makes sense to have it as an attribute on `mini-profiler`? Would the intention instead be to use something like `@NonceSource.GetNonce()`? 

As far as documentation / samples go, what is the story for "I want to inform miniprofiler about a nonce that comes from some piece of middleware?" The middleware shoves it in the context somewhere and it gets pulled in here?

Reply:
Correct - you'd pass in a unique value. Even putting say a `<Func<HttpContext, string>` is complicated due to the different primitives with .NET vs. .NET Core...maybe when we can drop .NET Full Framework support one day that gets easier since there's no common ancestor need.

Reply:
Oh sorry misread the second part - there's no built-in middleware story I can think of that makes sense here. The problem is generally you want MiniProfiler to profile the things you're doing and we have an order inversion problems with dependencies about the middleware doing it. Either direction we choose would be wrong for some subset of users. At least in v1, I'd want to leave it up to the users to handle their nonce generation.